examples/android/clientcache/app/build.gradle

```diff
@@ -49,11 +49,14 @@ dependencies {
     implementation 'com.android.support:appcompat-v7:27.0.2'
 
     // You need to build grpc-java to obtain these libraries below.
-    implementation 'io.grpc:grpc-okhttp:1.23.0-SNAPSHOT' // CURRENT_GRPC_VERSION
+    runtimeOnly 'io.grpc:grpc-okhttp:1.23.0-SNAPSHOT' // CURRENT_GRPC_VERSION
     implementation 'io.grpc:grpc-protobuf-lite:1.23.0-SNAPSHOT' // CURRENT_GRPC_VERSION
     implementation 'io.grpc:grpc-stub:1.23.0-SNAPSHOT' // CURRENT_GRPC_VERSION
     implementation 'javax.annotation:javax.annotation-api:1.2'
 
+    // You need to build SNAPSHOT version of grpc-android to obtain these library below.
```

Comment:
This comment will go to release versions, and they don't apply. It's better to add instruction of how to do it [here](https://github.com/grpc/grpc-java/tree/master/examples/android#install) instead.

Reply:
Should we also mention `AndroidChannelBuilder` is experimental there?

Reply:
Oh, AndroidChannelBuilder is still experimental? Then we shouldn't reference it in the example. We should consider stabilizing it.

Please don't encourage users to build SNAPSHOT versions. We can tell them how, but we should encourage them to check out the latest release's tag. That is a _far_ better experience.

Reply:
Actually `OkhttpChannelBuilder` and `NettyChannelBuilder` are experimental, and they are not mentioned as experimental in examples. So we don't need mention. However, `ManagedChannelBuilder` is stable, so I hesitate whether it's worth replacing the stable `ManagedChannelBuilder` with the experimental  `OkhttpChannelBuilder` in the example.

Reply:
Now that we have grpc-android published to Maven already, our Android examples should use it as the first-class citizen.

Reply:
We're changing helloworld. Helloword should _not_ use any experimental APIs. If they are doing something advanced, then experimental can be fine.

Reply:
This is Android's helloworld. I am wondering if we should recommend users always use `AndroidChannelBulider` under Android environment? (I was suppose to mention this in Android Dev Guide)

Reply:
We should not recommend users to always use an experimental API.